js/utils.js

```diff
@@ -1300,12 +1301,20 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
           field: field,
           useData: true
         }), function(item) {
+          var dates = records.map(function(obj) {
+            return moment(obj.data.Date);
+          });
+          var minDate = moment.min(dates).format(LOCAL_FORMAT);
```

Comment:
When there are invalid values, `moment.min()` will return the invalid value as the result. The same goes for `moment.max()`.

The invalid dates need to be excluded before calling `moment.min()` to avoid this problem.

![image](https://user-images.githubusercontent.com/290733/130028443-80e1e1c4-a954-40ae-a961-1d833a956e23.png)